### ISSUE TITLE
It's not Obvious what exactly is meant with macros.

### ISSUE THREAD

**Issue by tfo-inc** [MicrosoftDocs/OfficeDocs-DeployOffice#1242]
In the documentation, it says if a device has recently opened a macro file.
On all our devices, macros are blocked, but there are still some devices showing macros.

We tested with some macro files, but the devices don't show up.

We think it should be clarified what exactly is meant with recently opened a macro file.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: e28f21c4-9bdb-00cc-f18f-053f89dc4702
* Version Independent ID: 0f9e593e-f4d2-4d3a-6bc3-a436d3f615d2
* Content: [Overview of inventory in the Microsoft 365 Apps admin center - Deploy Office](https://learn.microsoft.com/en-us/deployoffice/admincenter/inventory#add-ins-insight)
* Content Source: [DeployOffice/admincenter/inventory.md](https://github.com/MicrosoftDocs/OfficeDocs-DeployOffice/blob/live/DeployOffice/admincenter/inventory.md)
* Service: **o365-proplus-itpro**
* GitHub Login: @nicholasswhite
* Microsoft Alias: **nwhite**

**Comment by tfo-inc** [comment]
Hello will anything happen here?

**Comment by DHB-MSFT** [comment]
@manoth-msft   Is this something you can help answer? Thanks.

**Comment by manoth-msft** [comment]
No, I don't have insights how the macro flag is calculated. I'll spin up an internal thread with the PM.

**Comment by officedocspr5** [comment]
To make it easier for you to submit feedback on articles on learn.microsoft.com, we're transitioning our feedback system from GitHub Issues to a new experience. 

As part of the transition, this GitHub Issue will be moved to a private repository. We're moving Issues to another repository so we can continue working on Issues that were open at the time of the transition. When this Issue is moved, you'll no longer be able to access it.

If you want to provide additional information before this Issue is moved, please update this Issue before December 15th, 2023.

With the new experience, you no longer need to sign in to GitHub to enter and submit your feedback. Instead, you can choose directly on each article's page whether the article was helpful. Then you can then choose one or more reasons for your feedback and optionally provide additional context before you select **Submit**.

Here's what the new experience looks like.

**Note**: The new experience is being rolled out across learn.microsoft.com in phases. If you don't see the new experience on an article, please check back later.

First, select whether the article was helpful:

![Image showing a dialog asking if the article was helpful with yes and no answers.](https://learn.microsoft.com/Office/media/internal/new-feedback-yes-no.jpg)

Then, choose at least one reason for your feedback and optionally provide additional details about your feedback:

| Article was helpful | Article was unhelpful |
|:--|:---|
|![Image showing a dialog asking how the article was helpful with several options.](https://learn.microsoft.com/Office/media/internal/new-feedback-helpful.jpg) | ![Image showing a dialog asking how the article wasn't helpful with several options.](https://learn.microsoft.com/Office/media/internal/new-feedback-unhelpful.jpg)|

Finally, select **Submit** and you're done!
